src/forms/checkbox-group.jsx

```diff
@@ -60,8 +69,8 @@ class CheckboxGroup extends MultipleChoice(Input) {
                 <label htmlFor={this.optionId(opt)} className="govuk-label govuk-checkboxes__label">{opt.label}</label>
                 { opt.hint && <span className="govuk-hint">{opt.hint}</span> }
                 {
-                  opt.reveal && (
-                    <div className={ classnames('govuk-reveal', { hidden: this.state && !this.hasValue(opt.value) }) }>
+                  opt.reveal && showReveal(opt)(
```

Comment:
This looks like it's missing an `&&`
```suggestion
                  opt.reveal && showReveal(opt) && (
```

Reply:
oops 🤦